gitpod-io/spring-petclinic
# Fix the detection of this project as a Maven module

## Description

Fix the detection of this project as a Maven module.

The important changes here are  `.idea/spring-petclinic.iml`, `.idea/modules.xml` (which references the first), and the `.idea/.gitignore` (when I deleted the .idea folder, IntelliJ recreated it with that file, and I added "/libraries/" to it).

## How to test

1. [Select IntelliJ IDEA as the preferred editor](https://gitpod.io/preferences) (It works for both Stable and Latest versions.)
2. Access https://gitpod.io/#https://github.com/gitpod-io/spring-petclinic/tree/vn/fix-maven-module-detection
3. Confirm if the workspace is initialized with Maven module correctly set.
    <img width="783" alt="Screenshot" src="https://user-images.githubusercontent.com/418083/189951975-1e9532e3-d66f-47fd-a761-c140e474a35e.png">
4. Confirm if, after the indexing finishes, there's no modified git file.
    <img width="401" alt="image" src="https://user-images.githubusercontent.com/418083/189952246-4c7e439f-a4d0-4a38-adc0-76f04c593fb9.png">
